Replace use of computeQuickBounds with getBBox.

kubaroth opened the pull request:
Addresses issue: Use GEO_Detail::getBBox Instead of GEO_Detail::computeQuickBounds #615
Signed-off-by: kubaroth <kubaroth@gmail.com>

Idclip commented:
Looks good, I take it we're not concerned with breaking existing behaviour as its classified as a bug? @danrbailey any idea why the tests are failing?

danrbailey commented:
> Looks good, I take it we're not concerned with breaking existing behaviour as its classified as a bug? @danrbailey any idea why the tests are failing?

I think the PR just got made at a moment when the CI was broken. @kubaroth - everything should be working now, so I'd recommend git merging the main master branch into this one and then pushing that commit and seeing if the tests resolve themselves. (I attempted to kick it a few days ago by switching the branch, but that didn't help). Change looks good to me otherwise.

kubaroth commented:
Thanks for the comments. I've rebased onto master and updated PR. 
Btw should I use develop or master in the future? Looks like develop has been a bit behind.

danrbailey commented:
> Thanks for the comments. I've rebased onto master and updated PR.
> Btw should I use develop or master in the future? Looks like develop has been a bit behind.

Great stuff, looks like that has fixed it! Use master - we experimented with changing our default branch to develop for a short period, but decided the extra process overhead wasn't worth it and discovered we can adjust the target branch instead which is a good alternative. Thanks for the reminder, I'll double-check no outstanding PRs have been made into develop, then delete that branch to avoid confusion.

danrbailey commented:
Thanks @kubaroth - I've just merged and submitted to be upstreamed (#702).

kubaroth commented:
Super! Thanks @danrbailey.
